src/storage/src/hummock/store/version.rs

```diff
@@ -684,7 +685,11 @@ impl HummockVersionReader {
             Sstable::hash_for_bloom_filter(dist_key.as_ref(), read_options.table_id.table_id())
         });
 
-        let full_key = FullKey::new(read_options.table_id, TableKey(table_key.clone()), epoch);
+        let full_key = FullKey::new_with_gap_epoch(
```

Comment:
Can you add some documentations on why we need to use `MAX_SPILL_OFFSET` and add a UT to cover the bug?

Reply:
How about UT? Are we planning to add it in later PR?

Reply:
Yes, UT will be added in https://github.com/risingwavelabs/risingwave/pull/14557.